### ISSUE TITLE
jsprintsetup is marked as incompatible with e10s

### ISSUE THREAD

**Issue by ghost** [edabg/jsprintsetup#27]
You can see this by visiting http://arewee10syet.com/  and searching for jsprintsetup

**Comment by h0jeZvgoxFepBQ2C** [comment]
yeah, except silent printing nearly nothing of the documented functionality works for me in the latest firefox... (hope I didnt do something wrong)
